### ISSUE TITLE
Issue 5 — Preprocesamiento: one-hot de etiquetas

### ISSUE THREAD

**Issue by Alaedddine718** [Alaedddine718/Deep-Learning_NV#6]
Issue 5 — Preprocesamiento: one-hot de etiquetas

Labels: preprocess, priority:high
Descripción: One-hot para 10 clases.
Checklist

 y_train_oh = keras.utils.to_categorical(y_train, 10) (igual para test).
DoD

 y_train_oh.shape == (50000,10).
